docs/concepts/projects/init.md

```diff
@@ -34,6 +34,10 @@ example-app
 └── pyproject.toml
 ```
 
+!!! tip
+
+    In versions of uv older than 0.6.0 `main.py` will instead be `hello.py`.
```

Comment:
Sorry, I know I suggested a tip at first :D

```suggestion
!!! note

    Prior to v0.6.0, uv created a file named `hello.py` instead of `main.py`.
```